docker-compose.yml

```diff
@@ -9,14 +9,19 @@ services:
       MONGO_INITDB_ROOT_PASSWORD: 'deep_thought'
       MONGO_INITDB_DATABASE: 'deep_thought'
     ports:
-      - 27017:27017
+      - 27018:27017
```

Comment:
Qual motivo por não mapear as portas padrões ?

Reply:
evitar conflito local com o ambiente já instalado.

Reply:
Acho que essa é uma característica do seu ambiente de desenvolvimento. Talvez seja bom deixarmos as portas padrões e então cada um que for implementar sua versão da App, altera as portas para o que faz mais sentido dentro do ambiente.

Reply:
done!